README.parallels

```diff
@@ -0,0 +1,19 @@
+vagrant-swift-all-in-one setup on Macbook M1 aarch64 processor with Parallels
+========================
+
+Host setup or installations 
```

Comment:
The is a space at the end of this line and on line 16. Only notice it when I do a `git diff HEAD~` when I was testing this out.